app/models/profile.rb

```diff
@@ -42,6 +42,10 @@ def work_model_preferences
     super || []
   end
 
+  def work_model_preferences=(values)
+    super(values.reject(&:empty?))
+  end
```

Comment:
Felt this was the easiest way to deal with how the checkboxes submit their data. If the user checks "remote" only, it'll come in as `["", "", "remote"]`.